packages/salesforcedx-vscode-core/src/commands/forceApexTestRun.ts

```diff
@@ -67,7 +67,7 @@ export class TestsSelector
     fileItems.push({
       label: nls.localize('force_apex_test_run_all_test_label'),
       description: nls.localize(
-        'force_apex_test_run_all_tests_desription_text'
+        'force_apex_test_run_all_tests_description_text'
```

Comment:
Sorry, I must be blind, what is the difference between this version and the previous one? Why is GitHub highlighting this?

Reply:
"description" was spelled incorrectly in the previous version.